docs/API/API-functions.md

```diff
@@ -358,27 +358,74 @@ int srt_bind(SRTSOCKET u, const struct sockaddr* name, int namelen);
 
 Binds a socket to a local address and port. Binding specifies the local network
 interface and the UDP port number to be used for the socket. When the local 
-address is a form of `INADDR_ANY`, then it's bound to all interfaces. When the 
-port number is 0, then the port number will be system-allocated if necessary.
+address is a wildcard (`INADDR_ANY` for IPv4 or `in6addr_any` for IPv6), then
+it's bound to all interfaces (although see `SRTO_IPV6ONLY` and additional
+information below for details about the wildcard address in IPv6).
+
+Binding is necessary for every socket to be used for communication. It can be
+bound implicitly, when the socket is being connected to a listening socket in
+the network using [`srt_connect`](#srt_connect) or a similar function that does
+the same, in which case the socket will be bound to a wildcard address of
+the IP version as specified in the `sockaddr::sa_family` field and port number
+0.  In other cases it must be bound explicitly using this function, or some
+other function that uses this functionality.
+
+When the port number is 0 (including the implicit binding case), then the port
+number will be system-allocated and can be determined after an explicit or
+implicit binding using [`srt_getsockname`](#srt_getsockname).
 
 This call is obligatory for a listening socket before calling [`srt_listen`](#srt_listen)
 and for rendezvous mode before calling [`srt_connect`](#srt_connect); otherwise it's 
 optional. For a listening socket it defines the network interface and the port where 
-the listener should expect a call request. In the case of rendezvous mode (when the
-socket has set [`SRTO_RENDEZVOUS`](API-socket-options.md#SRTO_RENDEZVOUS) to 
-true both parties connect to one another) it defines the network interface and port 
-from which packets will be sent to the peer, and the port to which the peer is 
-expected to send packets.
-
-For a connecting socket this call can set up the outgoing port to be used in the 
-communication. It is allowed that multiple SRT sockets share one local outgoing 
-port, as long as [`SRTO_REUSEADDR`](API-socket-options.md#SRTO_REUSEADDRS) 
-is set to *true* (default). Without this call the port will be automatically 
-selected by the system.
+the listener should expect a call request.
+
+In the case of rendezvous mode there's a pair of two endpoints (address-and-port
+specifications), let's name them A and B, and for both parties one endpoint is
+local and the other is remote one. The party, for which the A's address is
+local, should bind to the A address, and then connect to the B endpoint, and the
+other party the other way around. Both sockets must be set
+[`SRTO_RENDEZVOUS`](API-socket-options.md#SRTO_RENDEZVOUS) to *true* to make
+this connection possible.
+
+For a connecting socket this call is optional, but can be used to set up the
```

Comment:
Which call? The context is lost as if `SRTO_RENDEZVOUS` is being referred to, while it is actually `srt_bind`.

Reply:
The call to `srt_bind` is meant here. 
```suggestion
For a connecting socket the call to `srt_bind` is optional, but can be used to set up the
```